src/core_plugins/console/api_server/es_6_0/query.js

```diff
@@ -76,6 +77,36 @@ const SPAN_QUERIES = {
     __scope_link: '.span_within',
   },
 };
+const SPAN_QUERIES = {
+  ...SPAN_QUERIES_NO_FIELD_MASK,
+  field_masking_span: {
+    __template: {
+      query: {
+        SPAN_QUERY: {}
+      }
+    },
+    query: SPAN_QUERIES_NO_FIELD_MASK,
+    field: ''
+  }
+};
+
+const SPAN_MULTI_QUERIES = {
+  wildcard: {
+    __scope_link: '.wildcard'
```

Comment:
I was expecting this construction here to give me the template for the the inner `wildcard` query itself, much like how it works when the `wildcard` query is used at the top-level, but I wasn't able to do that.

![wildcard](https://user-images.githubusercontent.com/51061/40428458-356880b2-5e55-11e8-8bad-fec3964065dc.gif)

Same with `prefix`, `range`, and `regexp`.